docs/dev_guide/scala_dev_guide.md

```diff
@@ -38,9 +38,16 @@ Feathr is compiled using [SBT](https://www.scala-sbt.org/1.x/docs/Command-Line-R
 
 To compile, run
 ```
-sbt compile
+sbt assembly
 ```
 
+To compile with certain java version, run
+```
+sbt assembly -java-home "/Library/Java/JavaVirtualMachines/jdk1.8.0_282-msft.jdk/Contents/Home"
```

Comment:
Not sure why msft jdk is used here, i think for most people here should be adoptopenjdk-8.jdk